code/index.html

```diff
@@ -2,17 +2,131 @@
+      <section class="container"
+        <img src="/assets/header.JPG" alt="katze bild">
+          <div> 
+    
+            <h1> The Cat Working Space </h1>
+            <h2> Unter nicen Kollegen</h2>
+          </div>
+        </section>
+      </section>
+
+
+        <section class="box-container">
+
+          <section class="container">
+          <h3>Katzen, Kreativtiät und Kaffee</h3>
+           
+          <section class="boxes"> 
+          
+             <a href="#">
+               <div>
+                 <figure class="circles">
+                   <span></span>
+                   <span></span>
+                  </figure>
+                   <p> Hier reservieren </p>
+                   <h4> Dein Arbeitsplatz </h4> 
+               </div>
+               <img src="/assets/Reserviere deinen Arbeitsplatz.jpg" alt="Tisch";
+             </a>
+
+             <a href="#">
+              <div>
+                <figure class="circles">
+                  <span></span>
+                  <span></span>
+                 </figure>
+                  <p> Deine Kollegen</p>
+                  <h4> Raja & Ramen </h4> 
+              </div>
+              <img src="/assets/Katzen.JPG" alt="Katzen";
+            </a>
+
+            <a href="#">
+              <div>
+                <figure class="circles">
+                  <span></span>
+                  <span></span>
+                 </figure>
+                  <p> Hier geht's ab</p>
+                  <h4> Dein Büro </h4> 
+              </div>
+              <img src="/assets/Das Büro.jpg" alt="Büro";
+            </a>
+
+            <a href="#">
+              <div>
+                <figure class="circles">
+                  <span></span>
+                  <span></span>
+                 </figure>
+                  <p> News, News, News</p>
+                  <h4> Zum Newsletter </h4> 
+              </div>
+              <img src="/assets/Newsletter.jpg" alt="Newsletter";
+            </a>
+
+            <a href="#">
+              <div>
+                <figure class="circles">
+                  <span></span>
+                  <span></span>
+                 </figure>
+                  <p> Leute treffen - und Katzen</p>
+                  <h4> Events</Cat-Working-Space> </h4> 
+              </div>
+              <img src="/assets/Events.jpg" alt="Katzen";
+            </a>
+
+            <a href="#">
+              <div>
+                <figure class="circles">
+                  <span></span>
+                  <span></span>
+                 </figure>
+                  <p> All you need is </p>
+                  <h4> Cats & Coffee: die Karte </h4> 
+              </div>
+              <img src="/assets/Kaffee.jpg" alt="Katzen";
+            </a>
+           </section>
+       
+          </section>
+        </section>
+      <footer>
+        <p> © CAT WORKONG SPACE<br>
+        <a href="mailto:hi@catworkingspace.com">hi@catworkingspace.com</a></p>
+      </footer>
```

Comment:
You could achieve something similar without using the `br` element (which is not a semantic element and we want to try to keep our HTML as semantic as possible:
```
<footer>
  <p> © CAT WORKONG SPACE</p>
  <a href="mailto:hi@catworkingspace.com">hi@catworkingspace.com</a>
</footer>
```
If you want to use this, you might want to change the margin of the `p` and `a` in the CSS